mtp_api/apps/transaction/serializers.py

```diff
@@ -31,3 +35,48 @@ class Meta:
             'owner',
             'credited',
         )
+
+
+class CreateTransactionListSerializer(serializers.ListSerializer):
+
+    @transaction.atomic
+    def create(self, validated_data):
+        upload_counter = Transaction.objects.all().aggregate(
+            Max('upload_counter')
+        )['upload_counter__max'] or 1
```

Comment:
Do we risk a race condition here? Is it worth ditching the counter in favour of a UUID or something?